cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -546,9 +561,7 @@ def __repr__(self) -> str:
         return f"css.ParallelGates({component_gates_repr})"
 
 
-def parallel_gates_operation(  # pylint: disable=missing-raises-doc
-    *ops: cirq.Operation,
-) -> cirq.Operation:
+def parallel_gates_operation(*ops: cirq.Operation) -> cirq.Operation:
     """Given operations acting on disjoint qubits, constructs a single css.ParallelGates instance
     and applies it such that each operation's .gate is applied to its .qubits.
```

Comment:
Per dev guide, to fit 1-line format, something like:
```suggestion
    """Constructs a parallel gates operation.
    
    Given operations acting on disjoint qubits, constructs a single css.ParallelGates instance
    and applies it such that each operation's .gate is applied to its .qubits.
```